src/domain/interfaces/balances-api.manager.interface.ts

```diff
@@ -8,10 +8,17 @@ export interface IBalancesApiManager {
    * Each chain is associated with an implementation (i.e.: to a balances
    * provider) via configuration.
    *
+   * Note: if the Safe entity associated with the passed safeAddress cannot be retrieved
+   * by the TransactionApi for the input chain ID, then the Zerion API will be used.
+   *
    * @param chainId - the chain identifier to check.
+   * @param safeAddress - the safe address to check.
```

Comment:
```suggestion
   * Note: if the Safe entity associated with th safeAddress cannot be retrieved
   * from the TransactionApi for the chainId, then the ZerionApi will be used.
   *
   * @param chainId - the chain identifier to check.
   * @param safeAddress - the Safe address to check.
```

Reply:
I've taken your suggestion in a different commit, as I think there's a typo on it:
```
associated with th safeAddress
```

I've changed it in [32c52a2](https://github.com/safe-global/safe-client-gateway/pull/1615/commits/32c52a21ac656ee3375d237d82204c5a0fca7c61)